app/views/admin/users/show/profile/actions/_banish.html.erb

```diff
@@ -1,21 +1,14 @@
 <div id="banish-for-spam">
   <div class="flex flex-col gap-4">
-    <% if @user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago %>
-      <p>This action is irreversible.</p>
-      <p>Once banished, we will delete all content created by <%= @user.name %> and change their username to @spam_###.</p>
-      <p>Be careful with this action.</p>
-      <%= form_for(@user, url: banish_admin_user_path(@user), html: { method: :post, onsubmit: "return confirm('Are you sure? This action is irreversible. Banishing will delete all content by #{@user.name} and change their username to @spam_###.')", id: nil }) do %>
-        <button class="c-btn c-btn--primary c-btn--destructive">Banish User for spam</button>
-      <% end %>
-    <% elsif current_user.super_admin? || current_user.support_admin? %>
+    <% if current_user.super_admin? || current_user.support_admin? %>
       <p>This action is irreversible.</p>
       <p>Once banished, we will delete all content created by <%= @user.name %> and change their username to @spam_###.</p>
       <p>Be careful with this action.</p>
       <%= form_for(@user, url: banish_admin_user_path(@user), html: { method: :post, onsubmit: "return confirm('Are you sure? This action is irreversible. Banishing will delete all content by #{@user.name} and change their username to @spam_###.')", id: nil }) do %>
         <button class="c-btn c-btn--primary c-btn--destructive">Banish <%= @user.name %></button>
       <% end %>
     <% else %>
-      <div class="crayons-notice crayons-notice--danger">This is not a new user. Only Super Admins or Support Admins are allowed to banish <%= @user.name %>.</div>
```

Comment:
Related to the previous comment I. suspect we have "This is not a new user" because we only banish users if their account is created within the last a 100 days and they have not commented in the last 100 days. 

I definitely think we should account for that conditional within our copy so that admins are not confused on why they are not able to banish a user.

Perhaps this is something to confirm with product.

Reply:
Reverted! ✔️